eng/Version.Details.xml

```diff
@@ -125,6 +125,10 @@
       <Uri>https://github.com/nuget/nuget.client</Uri>
       <Sha>d2bb0de35242b04802c1a7856ebb7718f674565c</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Cli.CommandLine" Version="1.0.0-preview.19208.1" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@sfoslund in the merge conflict I just resolved, this was missing in main but present in 5.0.3xx. I chose to keep it.  Is it needed (ie did I mess up the merge)?

Reply:
This shouldn't be needed, this is the old command line package we used before switching to system.commandline

Reply:
Ok, removed. Now there are no differences anymore.  Do we usually close at that point or merge anyway to get the commits included?

Reply:
We usually close them. The bot will reopen when there are new commits and potentially real changes.

Reply:
ok, I'll close then.